ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -112,7 +112,9 @@ ${listenerText}
       return `wget -q https://releases.hashicorp.com/boundary-worker/0.11.2+hcp/boundary-worker_0.11.2+hcp_linux_amd64.zip ;\\
 sudo apt-get update && sudo apt-get install unzip ;\\
 unzip *.zip ;\\
-./boundary-worker server -config="${this.configFilePath || '<path>'}/pki-worker.hcl"`;
+./boundary-worker server -config="${
+        this.configFilePath || '<path>'
+      }/pki-worker.hcl"`;
```

Comment:
At first I was concerned that this changed the meaning, but it broke the lines within the JS interpolation.  I think I can factor out the value so that this remains on one line.

Reply:
Good point. I did go a head and confirm that it does not cause issues.
<img width="1065" alt="Screen Shot 2022-12-13 at 4 16 53 PM" src="https://user-images.githubusercontent.com/29386339/207456351-537c2580-1d6f-4bb5-b496-766b525635ab.png">